M2/Macaulay2/packages/Macaulay2Doc/overview_doc.m2

```diff
@@ -1,362 +1,238 @@
-	  "hypertext list format",
-	  document}
-     }
-document {
-     Key => "optional argument name documentation template",
- 	PRE ///document {
-     Key => optionName,
-     Headline => "one line description",
-     "There can be explanatory prose here in the form of a hypertext list.",
-     EXAMPLE {
-	  "m2code",
-	  "m2code",
-	  "m2code"
-	  },
-     "There can be explanatory prose here in the form of a hypertext list.",
-     Caveat => {"warning"}
-     }///,
-     SeeAlso => {"writing documentation",
-	  "hypertext list format",
-	  document}
-     }
-document {
-     Key => "optional argument documentation template",
- 	PRE ///document {
-     Key => [functionName, optionName],
-     Headline => "one line description",
-     Usage => "usage",
-     Inputs => {
-	  -- each input is a hypertext list
-	  },
-     Consequences => {
-          -- each effect is a hypertext list
-	  },
-     "There can be explanatory prose here in the form of a hypertext list.",
-     EXAMPLE {
-	  "m2code",
-	  "m2code",
-	  "m2code"
-	  },
-     "There can be explanatory prose here in the form of a hypertext list.",
-     Caveat => {"warning"}
-     }///,
-     SeeAlso => {"writing documentation",
-	  "hypertext list format",
-	  document}
-     }
-document {
-     Key => "overview documentation template",
- 	PRE ///document {
-     Key => "description of this page",
-     Headline => "one line description",
-     "There can be explanatory prose here in the form of a hypertext list.",
-     EXAMPLE {
-	  "m2code",
-	  "m2code",
-	  "m2code"
-	  },
-     "There can be explanatory prose here in the form of a hypertext list.",
-     Caveat => {"warning"}
-     }///,
-     SeeAlso => {"writing documentation",
-	  "hypertext list format",
-	  document}
-     }
-document {
-     Key => "package documentation template",
- 	PRE ///document {
-     Key => PACKAGENAME,
-     Headline => "one line description",
-     "There can be explanatory prose here in the form of a hypertext list.",
-     EXAMPLE {
-	  "m2code",
-	  "m2code",
-	  "m2code"
-	  },
-     "There can be explanatory prose here in the form of a hypertext list.",
-     Caveat => {"warning"}
-     Subnodes => {
-	  TO functionName1,
-	  TO functionName2 -- and so on
-	  }
-     }///,
-     SeeAlso => {"writing documentation",
-	  "hypertext list format",
-	  document}
-     }
+      There are a few stylistic conventions that should be noted. While not hard and fast rules,
+      keeping these stylistic conventions in mind makes for easier reading by users.
 
-undocumented {(SYNOPSIS, Sequence),(SYNOPSIS, Thing),(SYNOPSIS, List)}
+      @UL {
+          {"Lowercase is used for all titles, unless a proper noun is being used."},
+          {"Use ", TO TO, " to reference any Macaulay2 function, option, or variable occurring in the documentation as a hyperlink."},
+          {"If one needs to refer to the ", TT "i", "-th coefficient of some object, then use the format as given here."},
```

Comment:
What is this comment trying to say?  Why say "object" instead of "polynomial"?  Where is the "format" said to be "given here"?

Reply:
I'm not sure what it meant either. I presume it meant use `TT "i"`, but that seems cumbersome.

Reply:
I'll remove it.